Apps/WebClient/src/Server/Models/DependentModel.cs

```diff
@@ -26,29 +26,51 @@ public class DependentModel
         /// <summary>
         /// Gets or sets the hdid of the dependent.
         /// </summary>
-        [JsonPropertyName("hdid")]
-        public string HdId { get; set; } = string.Empty;
+        public DependentInformation DependentInformation { get; set; } = new DependentInformation();
 
         /// <summary>
-        /// Gets or sets the name of the dependent.
+        /// Gets or sets the owner of the hdid.
         /// </summary>
-        [JsonPropertyName("name")]
-        public string Name { get; set; } = string.Empty;
+        public string OwnerId { get; set; } = null!;
 
         /// <summary>
-        /// Gets or sets the Masked PHN of the dependent.
+        /// Gets or sets the hdid which has delegated access to the owner Id.
+        public string DelegateId { get; set; } = null!;
+
+        /// <summary>
+        /// Gets or sets the version.
         /// </summary>
-        [JsonPropertyName("maskedPHN")]
-        public string MaskedPHN { get; set; } = string.Empty;
+        public uint Version { get; set; }
```

Comment:
No documentation on this property, is it not warning you on web client?

Reply:
Not sure what you mean. These are the docs I see 
```
        /// <summary>
        /// Gets or sets the version.
        /// </summary>
```